src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -207,6 +221,28 @@ def get(self, request, organization):
         except (ValidationError, discover.InvalidSearchQuery) as exc:
             return Response({"detail": six.text_type(exc)}, status=400)
 
+        inbox_tab_cursor_results = None
+        if has_inbox:
+            inbox_tab_cursor_results = {}
+            if request.GET.get("query"):
+                inbox_tab_cursor_results[request.GET.get("query")] = (
+                    cursor_result.hits if cursor_result.hits else 0
+                )
+
+            for query in self.inbox_queries:
+                if query != request.GET.get("query"):
+                    _cursor_result, _query_kwargs = self._search(
+                        request,
+                        organization,
+                        projects,
+                        environments,
+                        {"count_hits": True, "date_to": end, "date_from": start},
```

Comment:
Does counting hits also fetch the results? Is there a way we could get counts without having to serialize all the results between snuba and sentry?